test/overall/simplify.mocha.coffee

```diff
@@ -56,6 +56,23 @@ describe "Simplify", ->
       ex2 = r(3).add('$y', 7)
       expect(ex1.simplify().toJS()).to.deep.equal(ex2.toJS())
 
+  describe 'fallback', ->
+    it "removes self if fallbackVal is null", ->
+      ex1 = $('x').fallback(null);
+      ex2 = $('x')
+      expect(ex1.simplify().toJS()).to.deep.equal(ex2.toJS())
+
+
+  describe 'power', ->
+    it "removes self if 1", ->
+      ex1 = r(3).power(1);
```

Comment:
this is not an interesting test. plywood would be smart enough to simplify `r(3).power(1)` without your help. in fact it can even do `r(3).power(5)` since these are all literals. Replace `r(3)` with `$('x')` for a meaningful test